.github/workflows/CI.yml

```diff
@@ -16,9 +16,14 @@ jobs:
           node-version: 12.x
       - name: Environment Information
         run: npx envinfo
-      - name: build it
+      - name: Build and Test on ubuntu and macOS
+        if: matrix.os != 'windows-latest'
         run: |
           cmake .
           cmake --build . --target check
-        env:
-          CI: true
+      - name: Build and Test on windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+          cmake -G "CodeBlocks - NMake Makefiles" .
```

Comment:
Out of curiosity, what does this line do?

Reply:
Current cmake generated config won't work for test. See https://github.com/gengjiawen/uvwasi/runs/258268519. I see clion use this config for  visual studio compiler. And the test works. So I am thinking change to this.